<launch>
    <node pkg="lidar2d_object_tracker" type="lidar2d_object_tracker" name="lidar2d_object_tracker"/>
    <param name="target_distance_tolerance" value="0.5"/> <!--m/s-->
    <param name="target_select_x_min" value="0.0"/>
    <param name="target_select_x_max" value="0.3"/>
    <param name="target_select_y_min" value="-0.2"/>
    <param name="target_select_y_max" value="0.2"/>
    <param name="target_select_duration" value="1"/> <!-- sec -->
</launch>